ext/gvl_tracing_native_extension/gvl_tracing.c

```diff
@@ -63,7 +63,7 @@ static VALUE gc_tracepoint = Qnil;
 void Init_gvl_tracing_native_extension(void) {
   rb_global_variable(&gc_tracepoint);
 
-  VALUE gvl_tracing_module = rb_define_module("GvlTracing");
+  VALUE gvl_tracing_module = rb_define_module("GvlTracingNativeExtension");
```

Comment:
You don't have to rename this. The module can have a mix of ruby and C methods.

If there is conflict in the name, I generally define them as private methods prefixed with an underscore, e.g. `_start`.